example/storybook/stories/components/primitives/Checkbox/controlledCheckbox.tsx

```diff
@@ -2,16 +2,21 @@ import React from 'react';
 import { Checkbox, CheckboxGroup, Text } from 'native-base';
 
 export default function ControlledCheckbox() {
-  const [groupValues, setGroupValues] = React.useState<Array<any>>([]);
+  const [groupValues, setGroupValues] = React.useState<Array<string>>([]);
 
   return (
-    <CheckboxGroup onChange={setGroupValues} value={groupValues}>
-      <Checkbox value="one">
+    <>
+      <Checkbox value="three" isChecked>
```

Comment:
@vidhi499 can we remove this isChecked example from here. I think this was just added while testing